content/applications/inventory_and_mrp/purchase/advanced/procurement_expenses_report.rst

```diff
@@ -0,0 +1,113 @@
+===========================
+Procurement expenses report
+===========================
+
+.. |RFQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |RFQs| replace:: :abbr:`RfQs (Requests for Quotation)`
+.. |POs| replace:: :abbr:`POs (Purchase Orders)`
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |per| replace:: procurement expenses report
+
+With the *Purchase* application, users can monitor procurement expenses over time. This report helps
+companies track and analyze spending, identify cost-saving opportunities, and ensure efficient
+budget management.
+
+Create procurement expenses report
+==================================
+
+To create a vendor costs report, first navigate to :menuselection:`Purchase app --> Reporting -->
+Purchase` to open the :guilabel:`Purchase Analysis` dashboard. By default, the dashboard displays a
+line chart overview of the :guilabel:`Untaxed Total` of POs (Purchase Orders) with a
+:guilabel:`Confirmation Date` for the current month, or of RFQs (Requests for Quotation) with a
+status of *Draft*, *Sent*, or *Cancelled*.
+
+Add filters and groups
+----------------------
+
+On the top-right, click the :icon:`oi-view-pivot` :guilabel:`(pivot)` icon to switch to pivot view.
```

Comment:
Personally, i think the report is the most useful for this report, especially because of the comparison factor (to be able to view the change from one date range to the next). I do not think you get the same impact in a pie chart, I think it is necessary to actually be able to view the numbers. I do mention that you can also view the information in a graph/chart form later in the doc, but I can add a note block specifying this.